recipes/numba/meta.yaml

```diff
@@ -0,0 +1,83 @@
+{% set name = "numba" %}
+{% set version = "0.36.1" %}
+{% set hash_type = "sha256" %}
+{% set hash_value = "a3b7540713c39f4d7a8d3b18cd49249c6356a35ee1ab5e15a89f7cf970004f0a" %}
+
+package:
+  name: '{{ name|lower }}'
+  version: '{{ version }}'
+
+source:
+  fn: '{{ name }}-{{ version }}.tar.gz'
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  '{{ hash_type }}': '{{ hash_value }}'
+
+build:
+  number: 0
+  script: python setup.py install  --single-version-externally-managed --record=record.txt
+
+requirements:
+  build:
+    - python
+    - setuptools
+    - llvmlite ==0.21.0
+    - numpy
+    - enum34
+    - singledispatch
+    - funcsigs
+  run:
+    - python
+    - llvmlite ==0.21.0
+    - numpy
+    - enum34
+    - singledispatch
+    - funcsigs
+
+test:
+  imports:
+    - numba
+    - numba.annotations
+    - numba.cuda
+    - numba.cuda.cudadrv
+    - numba.cuda.kernels
+    - numba.cuda.simulator
+    - numba.cuda.simulator.cudadrv
+    - numba.cuda.tests
+    - numba.cuda.tests.cudadrv
+    - numba.cuda.tests.cudadrv.data
+    - numba.cuda.tests.cudapy
+    - numba.cuda.tests.cudasim
+    - numba.cuda.tests.nocuda
+    - numba.datamodel
+    - numba.jitclass
+    - numba.npyufunc
+    - numba.pycc
+    - numba.rewrites
+    - numba.runtime
+    - numba.scripts
+    - numba.servicelib
+    - numba.targets
+    - numba.testing
+    - numba.tests
+    - numba.tests.npyufunc
+    - numba.typeconv
+    - numba.types
+    - numba.typing
+    - numba.unsafe
```

Comment:
Are there some tests that we can run?

Reply:
Should have been clearer. Was meaning could we run their test suite (or some part of it if it is too long)?

Reply:
Would be worth doing some of [these things]( https://github.com/numba/numba/blob/0.36.1/buildscripts/condarecipe.buildbot/meta.yaml#L58-L65 ).